packages/app/src/runner/injectBundle.ts

```diff
@@ -1,11 +1,9 @@
-export async function injectBundle (namespace: string) {
-  const src = `/${namespace}/runner/cypress_runner.js`
+import pDefer from 'p-defer'
 
-  const alreadyInjected = document.querySelector(`script[src="${src}"]`)
+export const dfd = pDefer()
 
-  if (alreadyInjected) {
-    return
-  }
+export function injectBundle (namespace: string) {
+  const src = `/${namespace}/runner/cypress_runner.js`
```

Comment:
Could skip declaring this variable and just assign the string directly to `script.src`, same as is done for `link.href` below.

Reply:
Nice, done